### PR TITLE
Add required steps to deploy v2 on Azure

### PR DESCRIPTION
**Pull request by amydoxym**
## Description
Add required steps to deploy v2 on Azure
.

### CODE REVIEW

**Review by gilclefes — APPROVED** [review]
looks good to me.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -172,3 +180,25 @@ steps:
...
+        sed -i 's/tag:.*/tag: $(IMAGE_TAGS)/' values.yaml
+
+  - task: CopyFiles@2
+    displayName: Copying source files
+    inputs:
+      SourceFolder: "./"
+      Contents: "*.yaml"
+      TargetFolder: "$(Build.ArtifactStagingDirectory)"
+
+  - task: PublishBuildArtifacts@1
```
Comment (shejasabin) [inline azure-pipelines.yml:198]: hey @amydoxym aren't we pushing already [here](https://github.com/one-acre-fund/fineract-frontend-v2/blob/9496bace2fc8b8fb76f0a4dd00de928a9df238bb/azure-pipelines.yml#L177)

Comment (amydoxym) [reply azure-pipelines.yml:198]: @shejasabin Yes, we are. This step is  to publish all the Azure build package. The step you mention is to builld and push the image on the registry

**Review by shejasabin — APPROVED** [review]

**Review by gilclefes — APPROVED** [review]
looks good to me

**Pull request merged**